JS/Lesson26/onder/index.html

```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <title>Posts</title>
-  <link rel="stylesheet" type="text/css" href="styles.css">
+    <meta charset="UTF-8">
+    <title>Mocked Posts</title>  
+    <link rel="stylesheet" type="text/css" href="./styles.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <a href="./createPost/create.html">Create Post</a>
+    <a href="./EditPost/edit.html">Edit Post</a>
```

Comment:
This url should not be on the main page, it should only be available for each individual post so that we can get a post id